src/InfoBoxes/Content/Engine.cpp

```diff
@@ -43,12 +43,14 @@ void
 UpdateInfoBoxContentRPM(InfoBoxData &data) noexcept
 {
   const NMEAInfo &basic = CommonInterface::Basic();
+  const DeviceConfig &device_settings = CommonInterface::GetSystemSettings().devices[0];
```

Comment:
This is completely wrong. You can easily see why by answering the question "Why 0?"

Reply:
> the InfoBox should not do calculations

Okay, so where to put the code which does the multiplication?